docs/user-guide/jobs.md

```diff
@@ -0,0 +1,256 @@
+
+A Container in a Pod may fail for a number of reasons, such as because the process in it exited with
+a non-zero exit code, or the Container was killed for exceeding a memory limit, etc.  If this
+happens, and the `.spec.template.containers[].restartPolicy = "OnFailure"`, then the Pod stays
+on the node, but the Container is re-run.  Therefore, your program needs to handle the the case when it is
+restarted locally, or else specify `.spec.template.containers[].restartPolicy = "Never"`.
+See [pods-states](pod-states.md) for more information on `restartPolicy`.
+
+An entire Pod can fail can also fail, for a number of reasons, such as when the pod is kicked off
+the node (node is upgraded, rebooted, delelted, etc.), or if a container of the Pod fails and the
+`.spec.template.containers[].restartPolicy = "Never"`.  When a Pod fails, then the Job controller
+starts a new Pod.  Therefore, your program needs to handle the case when it is restarted in a new
+pod.  In particular, it needs to handle temporary files, locks, incomplete output and the like
+caused by previous runs.
+
+Note that even if you specify `.spec.parallelism = 1` and `.spec.completions = 1` and
+`.spec.template.containers[].restartPolicy = "Never"`, the same program may
+sometimes be started twice.
+
+If you do specify `.spec.parallelism` and `.spec.completions` both greater than 1, then there may be
+multiple pods running at once.  Therefore, you pods must also be tolerant of concurrency.
+
+## Alternatives to Job
+
+### Bare Pods
+
+When a the node that a pod is running on reboots or fails, the pod is terminated
+and will not be restarted.  However, a Job will create new pods to replace terminated ones.
+For this reason, we recommend that you use a job rather than a bare pod, even if your application
+requires only a single pod.
+
+### Replication Controller
+
+Jobs are complementary to [Replication Controllers](replication-controller.md).
+A Replication Controller manages pods which are not expected to terminate (e.g. web servers), and a Job
+manages pods that are expected to terminate (e.g. batch jobs).
+
+As discussed in [life of a pod](pod-states.md), `Job` is *only* appropriate for pods with
+`RestartPolicy` equal to o `OnFailure` or `Never`.  (Note: If `RestartPolicy` is not set, the default value is `Always`.)
+
+## Caveats
+
+Job is part of the experimental API group, so it is not subject to the same compatibility
+guarantees as objects in the main API.  It may not be enabled.  Enable by setting
+`--runtime-config=experimental/v1` on the apiserver.
+
+In general, pods which are created do not disappear until someone destroys them.  This might be a
+human or a `Jobs`, or another controller.  The only exception to this rule is that pods with a
+`PodPhase` of `Succeeded` or `Failed` for more than some duration (determined by the master) will
+expire and be automatically reaped.  But that automatic reaping interferes with completion counting.
```

Comment:
I'm not sure what you mean with the last statement. Maybe that should be:
"... be automatically reaped by the automatic reaping interfaces with completion counting." ?